### PR TITLE
Added maven wrapper and renamed resource folder

### PR DESCRIPTION
**Pull request by esend7881**
I noticed that with the capitalized Resource folder, maven install did not pick up on the files in that folder. By naming it with proper capitalization, maven works just fine.

(It is possible people who ran maven through intellj or eclipse did not notice that due to the IDE doing work)

### CODE REVIEW

File: mvnw [hunk 1]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (peterjhill) [inline mvnw:1]: this should not be checked in either, please.

File: .mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,3 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```
Comment (peterjhill) [inline .mvn/wrapper/maven-wrapper.properties:1]: .mvn shouldn't be checked in

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
+/*
```
Comment (peterjhill) [inline .mvn/wrapper/MavenWrapperDownloader.java:1]: .mvn should not be checked in

File: mvnw.cmd [hunk 4]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (peterjhill) [inline mvnw.cmd:1]: .mvn/* should not be checked in

**Comment by esend7881** [comment]
Many projects do come with a maven wrapper (so the user doesn't need a global install of maven). In fact, all gradle projects have a gradle wrapper. Are you sure you don't want them included?

**Comment by peterjhill** [comment]
ah, okay.. I'm constrained by what I've seen. I have not used gradle yet and have always needed to have maven installed. 

I'll send an email out to the people at Juniper that helped me get my PR merged and let them know that this is here.

**Comment by esend7881** [comment]
@peterjhill - cool, no problem. You can use the maven wrapper by doing a `./mvnw install` instead of the global `mvn install`

A caveat, you must have the `JAVA_HOME` variable set up and pointing to your jdk folder. 

It's pretty neat and in the end, allows people to only need Java installed instead of Java + (your required version of) Maven.

**Pull request merged**